sysutils/stressdrive/Portfile

```diff
@@ -5,7 +5,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           xcode 1.0
 
-github.setup        rentzsch stressdrive 1.3.2
+github.setup        rentzsch stressdrive 1.4
 revision            1
```

Comment:
revision field should be reset to 0 when updating the version.

Reply:
done